src/constants.ts

```diff
@@ -1,2 +1,4 @@
+import { ethers } from "ethers";
+
 export const UI_UPDATE_PERIOD = 400; // ms
-export const PAYMENT_AMOUNT = 1;
+export const PAYMENT_AMOUNT = ethers.parseEther("1");
```

Comment:
This is large, but I just wanted an amount I could easily see the difference on.

Reply:
This PR adds an input box to set the `amount` so it should be easier to use: https://github.com/statechannels/SCBridge-Wallet/pull/113

Reply:
I've rebased and includes these changes.